src/components/organisms/Onboarding/StepBody.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from 'react'
+import styles from './StepBody.module.css'
+import Markdown from '../../atoms/Markdown'
+import { ReactNode } from 'react-markdown'
+
+export default function StepBody({
+  body,
+  image,
+  children
+}: {
+  body: string
+  image?: string
+  children?: ReactNode
```

Comment:
thought(non-blocking): I think using children here does potentially introduce some confusion. It communicates that any elements can be passed, even though it is only used to display the steps actions.

I feel like the better approach would be to pass the actions to the component, like:
```jsx
export default function StepBody({
  body,
  image,
  actions
}: {
  body: string
  image?: string
  actions?: IStepAction[]
})
```

and then display `StepAction`s for each action passed